hack/verify-staging-imports.sh

```diff
@@ -25,8 +25,20 @@ kube::golang::setup_env
 
 
 for dep in $(ls -1 ${KUBE_ROOT}/staging/src/k8s.io/); do
-	if go list -f {{.Deps}} ./vendor/k8s.io/${dep}/... | tr " " '\n' | grep k8s.io/kubernetes | grep -v 'k8s.io/kubernetes/vendor' | LC_ALL=C sort -u | grep -e "."; then
-		echo "${dep} has a cyclical dependency"
+	if go list -f "{{.Deps}}" ./vendor/k8s.io/${dep}/... | tr " " '\n' | grep k8s.io/kubernetes | grep -v 'k8s.io/kubernetes/vendor' | LC_ALL=C sort -u | grep -qe "."; then
+		echo "${dep} has a cyclical dependency:"
+		echo
+		go list -f '{{.ImportPath}} {{.Deps}}' ./vendor/k8s.io/${dep}/... |
```

Comment:
Surely there is a cleaner and more approachable way to do this using the Golang AST API... @deads2k you owe me lunch for making this the first thing I looked at this morning